serverlessworkflow/modules/ROOT/pages/eventing/consume-produce-events-with-knative-eventing.adoc

```diff
@@ -9,178 +9,184 @@
 
-By default, the application will generate a link:{kubernetes_probes_url}[health check probe] to verify if the Knative platform injected the `K_SINK` variable and it's valid. If the variable is not present, the pod won't be in ready state to receive requests.
+The following table lists the health check probe configuration property:
 
-.Health Check Probe configuration
+.Health check probe configuration property
+[cols="30%,30%,40%", options="header"]
 |===
-|Property|Default|Description
+|Property|Default value|Description
 
-|org.kie.kogito.addons.knative.health_enabled
-|true
-|Whether to enable the health check to verify if the `K_SINK` variable has been injected in the environment.
+|`org.kie.kogito.addons.knative.health_enabled`
+|`true`
+|This property indicates if the health check is enabled to verify that the `K_SINK` variable is injected into the environment.
 
 |===
+--
 
-=== Knative Target Sink Generation Configuration
+Knative target sink generation configuration::
++
+--
+The Kogito Knative Eventing add-on generates a few Knative objects during build time. By default, the add-on generates a link:{knative_eventing_doc_url}/broker[Knative Broker] named `default` if the workflow application is acting as an event source.
 
-The Kogito Knative Eventing add-on generates a few Knative objects during build time. By default, it generates a link:{knative_eventing_doc_url}/broker[Knative Broker] named `default` if the workflow application is an Event Source.
+The following table lists the configuration properties related to Knative sink generation:
 
-.Knative Sink generation configuration
+.Knative sink generation configuration properties
+[cols="30%,30%,40%", options="header"]
 |===
-|Property|Default|Description
+|Property|Default value|Description
 
-|org.kie.kogito.addons.knative.auto_generate_broker
+|`org.kie.kogito.addons.knative.auto_generate_broker`
 |true
-|Whether the add-on should generate a default Knative Broker in memory to sink and dispatch the messages. Turn this property to `false` in case you already have a broker installed in your namespace rather than the default one. Note that you can use `org.kie.kogito.addons.knative.eventing.sink.*` to configure your custom Sink. If not defined, this auto generated Broker will work as the Sink.
+|This property indicates if the Kogito Knative Eventing add-on generates a default Knative Broker in memory to sink and dispatch the messages. Set this property to `false` in case a broker is already installed in your namespace instead of the default one. Note that you can use `org.kie.kogito.addons.knative.eventing.sink.*` property to configure your custom sink. If this property is not set, then the auto-generated broker works as a sink.
```

Comment:
```suggestion
|This property indicates if the Kogito Knative Eventing add-on generates a default Knative Broker in memory to sink and dispatch the messages. Set this property to `false` in case a broker is already installed in your namespace. Note that you can use `org.kie.kogito.addons.knative.eventing.sink.*` property to configure your custom sink. If this property is not set, then the auto-generated broker works as a sink.
```

Reply:
Done